dev_scripts/convert_params.py

```diff
@@ -0,0 +1,93 @@
+"""Script converting legacy param and json parameter files to hdf5.
```

Comment:
Is this PR still relevant. I see: https://github.com/jonescompneurolab/hnn-core/tree/master/dev_scripts already in `main` ? Or does it need a rebase?

Reply:
It's currently held up by #733. Will be rebased after that one is merged.